lxc/completion.go

```diff
@@ -1174,13 +1174,14 @@ func (g *cmdGlobal) cmpStoragePoolWithVolume(toComplete string) ([]string, cobra
 
 	var results []string
 	for _, volume := range volumes {
-		results = append(results, fmt.Sprintf("%s/%s", pool, volume))
+		volName, _ := parseVolume("volume", volume)
+		results = append(results, fmt.Sprintf("%s/%s", pool, volName))
```

Comment:
I know the code base is filled with this `fmt.Sprintf("%s/%s")` or other simple concatenation of strings but those are less efficient than a simple `+` operation when fancy conversion/formatting isn't required.

Reply:
You're right! I'll update that :)